API/DTOs/ProgressDto.cs

```diff
@@ -21,7 +21,7 @@ public class ProgressDto
     /// </summary>
     public string BookScrollId { get; set; }
     /// <summary>
-    /// Last time the progress was synced from UI or external app
+    /// Last time in Utc the progress was synced from UI or external app
     /// </summary>
-    public DateTime LastModified { get; set; }
+    public DateTime LastModifiedUtc { get; set; }
```

Comment:
Do not cause breaking changes. Add the additional field, do not break my implementations

Reply:
this is not a breaking change IMHO since this field was actually not filled with any data.

Reply:
ProgressDto is built via AutoMapper and since AppUserProgress has this field, the data will be mapped. So it is a breaking change as you're changing what is being mapped and hence the UI which may rely on that field is now not getting the data.

Reply:
ok, I understand